ledger/src/shred/merkle.rs

```diff
@@ -364,7 +401,11 @@ impl ShredCode {
         Ok(Self::SIZE_OF_HEADERS + Self::capacity(proof_size, /*chained:*/ true, resigned)?)
     }
 
-    fn chained_merkle_root(&self) -> Result<Hash, Error> {
+    fn get_chained_merkle_root_offset(proof_size: u8, resigned: bool) -> Result<usize, Error> {
```

Comment:
similarly here, `fn chained_merkle_root_offset` should call into this function instead of duplicating last line.